src/main/plugins/currency-converter-plugin/currency-converter-plugin.ts

```diff
@@ -65,6 +65,12 @@ export class CurrencyConverterPlugin implements ExecutionPlugin {
         return this.clipboardCopier(searchResultItem.executionArgument);
     }
 
+    thousands_separators(num: string) {
+        const numParts = num.toString().split(".");
+        numParts[0] = numParts[0].replace(/\B(?=(\d{3})+(?!\d))/g, this.config.separator);
+        return numParts.join(".");
+    }
```

Comment:
I think this function can be extracted from the plugin into some kind of helper area. And please also add unit tests for this.